docs/source/models/resnet.rst

```diff
@@ -6,6 +6,12 @@ ResNet
 The ResNet model is based on the `Deep Residual Learning for Image Recognition
 <https://arxiv.org/abs/1512.03385>`_ paper.
 
+.. note::
+    Bottleneck in torchvision places the stride for downsampling at 3x3 convolution(self.conv2)
+    while original implementation places the stride at the first 1x1 convolution(self.conv1)
+    according to "Deep residual learning for image recognition"https://arxiv.org/abs/1512.03385.
```

Comment:
This hyperlink will not render correctly. Given we already have a link to the paper few lines above, perhaps we can just say:
```suggestion
    according to the paper.
```